site/src/xServices/createWorkspace/createWorkspaceXService.ts

```diff
@@ -246,12 +235,6 @@ const checkCreateWSPermissions = async (organizationId: string) => {
   }) as Promise<Record<keyof typeof permissionsToCheck, boolean>>;
 };
 
-export const watchGitAuthRefresh = (callback: () => void) => {
-  const bc = new BroadcastChannel(REFRESH_GITAUTH_BROADCAST_CHANNEL);
-  bc.addEventListener("message", callback);
-  return bc;
-};
```

Comment:
This is a little different than that case, because it has a timeout where it will stop polling until the user interacts again, so this definitely won't be an "increases load by xx%" type of thing. I get the appeal of `BroadcastChannel`, but I've always found that it ends up being fragile for stuff like this.